bindings/nodejs/package.json

```diff
@@ -18,7 +18,7 @@
         "prebuild-windows-arm64": "prebuild --runtime napi --target 6 --prepack 'yarn run napi-build-windows-arm64' --strip --arch arm64",
         "napi-build-windows-arm64": "napi build --cargo-flags=--profile=production --target aarch64-pc-windows-msvc",
         "rebuild": "node scripts/build && tsc && node scripts/strip.js",
-        "install": "prebuild-install --runtime napi --tag-prefix=iota-sdk-nodejs-v && tsc || npm run rebuild",
+        "install": "prebuild-install --runtime napi --tag-prefix=iota-sdk-nodejs-v && tsc || yarn rebuild",
```

Comment:
Why? Just making sure this won't have side effects during release process etc

Reply:
well we have yarn in the rest of the commands, figured we should at least be consistent. Wouldnt change anything to the CI, only for people without yarn installed it would error ofcourse